core/modules/widgets/widget.js

```diff
@@ -432,28 +439,38 @@ Widget.prototype.assignAttributes = function(domNode,options) {
 				namespace = "http://www.w3.org/1999/xlink";
 				name = name.substr(6);
 			}
-			// Handle styles
-			if(name.substr(0,6) === "style." && name.length > 6) {
-				domNode.style[$tw.utils.unHyphenateCss(name.substr(6))] = value;
-			} else {
-				// Setting certain attributes can cause a DOM error (eg xmlns on the svg element)
-				try {
-					domNode.setAttributeNS(namespace,name,value);
-				} catch(e) {
-				}
+			// Setting certain attributes can cause a DOM error (eg xmlns on the svg element)
+			try {
+				domNode.setAttributeNS(namespace,name,value);
+			} catch(e) {
 			}
 		}
 	}
-	// Not all parse tree nodes have the orderedAttributes property
-	if(this.parseTreeNode.orderedAttributes) {
```

Comment:
Are we absolutely sure we do not need to respect the `orderedAttributes` order if it exists? I cannot think of any reason we do need to care about it but just wanted to flag this as something to review. Let Widget is the only one I can think of where order matters and it isn't impacted by this change.